<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KasaHareketEkleDialog</class>
 <widget class="QDialog" name="KasaHareketEkleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>310</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>260</width>
    <height>310</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>260</width>
    <height>310</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Monospace</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Kasa Hareketi Gir</string>
  </property>
  <widget class="QPushButton" name="KaydetpushButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>280</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Kaydet</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/ui/check.png</normaloff>:/images/ui/check.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="iptalpushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>İptal</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/ui/remove.png</normaloff>:/images/ui/remove.png</iconset>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>241</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Tarih</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDateEdit" name="tarihdateEdit">
      <property name="displayFormat">
       <string>dd.MM.yyyy</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Hareket</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="HareketcomboBox">
      <item>
       <property name="text">
        <string>GİRİŞ</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ÇIKIŞ</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>BANKAYA VİRMAN</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>İşlem Yapan</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="Kullanicilabel">
      <property name="text">
       <string>Kullanici</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Açıklama</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPlainTextEdit" name="AciklamaplainTextEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Evrak No</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="EvrakNolineEdit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Tutar</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QDoubleSpinBox" name="tutardoubleSpinBox">
      <property name="maximum">
       <double>9999999.990000000223517</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tarihdateEdit</tabstop>
  <tabstop>HareketcomboBox</tabstop>
  <tabstop>AciklamaplainTextEdit</tabstop>
  <tabstop>EvrakNolineEdit</tabstop>
  <tabstop>tutardoubleSpinBox</tabstop>
  <tabstop>KaydetpushButton</tabstop>
  <tabstop>iptalpushButton</tabstop>
 </tabstops>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
